.mergify.yml

```diff
@@ -2,7 +2,8 @@
 pull_request_rules:
   - name: automatic merge and new release from cloudalchemybot
     conditions:
-      - "status-success=Travis CI - Pull Request"
+      - "status-success=ci/circleci: test-2.10"
```

Comment:
Minor nit, it looks like Mergify has renamed this to `check-success`.

https://docs.mergify.io/conditions/#conditions